owasp-top10-2017-apps/a3/insecure-go-project/deployments/docker-compose.yml

```diff
@@ -21,6 +21,11 @@ services:
             - mongodb
         external_links:
             - mongodb:mongodb
+        environment:
+            - MONGO_DB_NAME=insecure_go_project
+            - MONGO_HOST=mongodb
+            - MONGO_USER=u_insecure_go_project
+            - MONGO_PASSWORD=svGX8SViufvYYNu6m3Kv
```

Comment:
Although you have removed these sensitive data from `config.yml`, you have just added them again here in this file. 😞

Reply:
Awesome, @Chavao !